transports/azure-storage-queues/native-integration_sending_asqn_[9,12].partial.md

```diff
@@ -0,0 +1,15 @@
+### Custom envelope unwrapper
```

Comment:
This seems to be the same as the paragraph above or am I not seeing the difference?

Reply:
one less sentence in the warning

Reply:
I see it now - wouldn't it be the same in both though it we are backporting the change?